module/solutions/helper.js

```diff
@@ -1506,5 +1512,126 @@ module.exports = class SolutionsHelper {
     });
 }
 
+    /**
+    * Add entity to solution.
+    * @method
+    * @name addEntityToSolution
+    * @param {String} solutionId - solution id.
+    * @param {Array} entityIds - Entity ids.
+    * @returns {String} - message.
+    */
+
+   static addEntityToSolution(solutionId,entityIds) {
+    return new Promise(async (resolve, reject) => {
+        try {
+  
+          let responseMessage = messageConstants.apiResponses.ENTITIES_UPDATED;
+          
+          let solutionQuery = {
+            isReusable : false
+          };
+
+          if( gen.utils.isValidMongoId(solutionId) ) {
+            solutionQuery["_id"] = solutionId;
+          } else {
+            solutionQuery["externalId"] = solutionId;
+          } 
+
+          let solutionDocument = 
```

Comment:
@aman-tunerlabs - This query can be avoided right, just find and update if matching id or external id and entityType etc